owners/src/parser.js

```diff
@@ -138,16 +139,16 @@ class OwnersParser {
    * Parse a list of owners.
    *
    * @private
-   * @param {!string} ownersPath OWNERS.yaml file path (for error reporting).
+   * @param {!string} ownersPath OWNERS file path (for error reporting).
    * @param {string[]} ownersList list of owners.
    * @return {OwnersParserResult<Owner[]>} list of owners' usernames.
```

Comment:
Maybe add an [`@deprecated`](https://jsdoc.app/tags-deprecated.html) annotation to this and all other yaml parsing functions?